ibm/data_source_ibm_dl_port.go

```diff
@@ -0,0 +1,79 @@
+package ibm
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceIBMDirectLinkPort() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceIBMDirectLinkPortRead,
+		Schema: map[string]*schema.Schema{
+			dlPortID: {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Port ID",
+			},
+			dlCount: {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: "Count of existing Direct Link gateways in this account on this port",
+			},
+			dlLabel: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Port Label",
+			},
+			dlLocationDisplayName: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Port location long name",
+			},
+			dlLocationName: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Port location long name",
+			},
+			dlProviderName: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Port location long name",
```

Comment:
description is same for all dlLocationDisplayName, dlLocationName, dlLocationName